TestAssets/TestProjects/MSBuildTestApp/project.json

```diff
@@ -11,6 +11,10 @@
   "runtimes": {
     "win7-x64": {},
     "win7-x86": {},
+    "win81-x64": {},
```

Comment:
It's the nuget targets which are failing. My guess is they don't understand rid fallback.